src/components/checkboxes/_checkbox.scss

```diff
@@ -30,6 +30,16 @@ $checkbox-padding: 11px;
     &:checked {
       background-size: 14px;
     }
+
+    &:disabled {
+      border: 2px solid $color-grey-35;
```

Comment:
Could you assign a variable to this colour in `_colors.scss`? e.g. `$color-borders-disabled`

Reply:
Added variables under `// Panels and status` as disabled felt like status but happy to move to a better location